10LessonTen/Checkpoint2/css/style.css

```diff
@@ -0,0 +1,281 @@
+body {
+  margin: 0;
+  font-family: 'Open Sans', sans-serif;
+}
+
+#navigation {
+  height:40px;
+  background-color: #d0285b;
+  margin: 0;
+}
+
+#navtitle {
+  font-size: 20px;
+  font-weight: 400;
+  padding: 0;
+  display:inline-block;
+  margin: 7px 0 0 5px;
+  color: white;
+}
+
+ul {
+  list-style-type: none;
+  width: 50%;
+  height: 40px;
+  margin: 0;
+  padding: 0;
+  display:inline-block;
+  float: right;
+  margin-top: 12px;
+}
+
+ul * {
+  list-style-type: none;
+  text-decoration: none;
+  color: white;
+  font-weight: 100;
+  font-size: 12px;
+}
+
+li {
+  float: right;
+  padding-right:20px;
+}
+
+/*TITLE CARD STARTS HERE */
+
+#titlecard {
+  background-image: url(../img/kangaroo1.jpg);
+  width: 100%;
+  height:600px;
+  padding:0;
+  margin: 0;
+  box-sizing: border-box;
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+
+#titletext {
+  box-sizing: border-box;
+  margin: 0;
+  text-align: center;
+  padding-top: 17%;
+  line-height: auto;
+  color:white;
+}
+
+#titlecard h1 {
+  font-size: 3.5em;
+  line-height: auto;
+  font-weight: 100;
+  margin:0;
+}
+
+#titlecard h2 {
+  font-weight: 100;
+  font-size: 1.5em;
+  margin:0;
+}
+
+.button {
+  width:130px;
+  height:40px;
+  background-color: white;
+  font-weight: 700;
+  margin: 1% auto;
+  border-radius: 15px;
+  text-align: center;
+  padding-top: 10px;
+  box-sizing: border-box;
+  font-size: 1em;
+}
+
+.button * {
+  text-decoration: none;
+  color: #970f2b;
+}
+
+/* SECTION ONE STARTS HERE */
+
```

Comment:
it may be better to have one generic "section"  class that has all your general rules and then get more specific when you need to.